ci/azure-pipelines-template.yml

```diff
@@ -47,14 +47,37 @@ jobs:
     displayName: Clippy
     condition: ne( variables['rustup_toolchain'], 'nightly' )
 
-  - ${{ if ne(parameters.name, 'Windows') }}:
+  - ${{ if eq(parameters.name, 'Linux') }}:
+    - script: |
+        cargo run --bin pyoxidizer -- init --pip-install cryptography ~/pyapp
+        cat ci/pyapp.py | cargo run --bin pyoxidizer -- run ~/pyapp
+      displayName: Build Oxidized Application
+
+  - ${{ if eq(parameters.name, 'macOS') }}:
+    - script: brew install openssl@1.1
+      displayName: 'Install OpenSSL'
+
+  - ${{ if eq(parameters.name, 'macOS') }}:
     - script: |
-        cargo run --bin pyoxidizer -- init --python-code 'print("hello, world")' ~/pyapp
-        cargo run --bin pyoxidizer -- run ~/pyapp
+        export CPPFLAGS="-I$(brew --prefix openssl@1.1)/include"
+        export LDFLAGS="-L$(brew --prefix openssl@1.1)/lib"
+        cargo run --bin pyoxidizer -- init --pip-install cryptography ~/pyapp
+        cat ci/pyapp.py | cargo run --bin pyoxidizer -- run ~/pyapp
       displayName: Build Oxidized Application
 
+  - ${{ if eq(parameters.name, 'Windows') }}:
+    - script: choco install openssl
+      displayName: 'Install OpenSSL'
+
+  - ${{ if eq(parameters.name, 'Windows') }}:
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '3.7'
```

Comment:
this is necessary because the windows job needs a functional Python, mostly due to https://github.com/indygreg/python-build-standalone/issues/22

https://github.com/indygreg/python-build-standalone/issues/19 is also problematic, but it can be worked around by using `virtualenv` instead of `venv`.  Currently the code in the PR does use `virtualenv` because I was initially attempting to use the standalone python, but cffi put a halt to that plan, so I am going to rip that out so that all OS use `venv`, simplifying a few areas of the changes.